### PR TITLE
more information in warning about widget not loading

### PR DESCRIPTION
**Pull request by amunger**
For #

-   [x] Pull request represents a single change (i.e. not fixing disparate/unrelated things in a single PR).
-   [x] Title summarizes what is changing.
-   [x] Has a [news entry](https://github.com/Microsoft/vscode-jupyter/tree/main/news) file (remember to thank yourself!).
-   [x] Appropriate comments and documentation strings in the code.
-   [ ] Has sufficient logging.
-   [ ] Has telemetry for enhancements.
-   [ ] Unit tests & system/integration tests are added/updated.
-   [ ] [Test plan](https://github.com/Microsoft/vscode-jupyter/blob/main/.github/test_plan.md) is updated as appropriate.
-   [ ] [`package-lock.json`](https://github.com/Microsoft/vscode-jupyter/blob/main/package-lock.json) has been regenerated by running `npm install` (if dependencies have changed).

### CODE REVIEW

**Comment by amunger** [comment]
This change will make it so we no longer ask the user to enable cdns if they are already enabled. Also adds more information to the warning message when we fail to download the widget (new message on top, previous message below):
image

File: package.nls.json [hunk 1]
```diff
@@ -489,7 +489,7 @@
     "DataScience.enableCDNForWidgetsSettingHtml": "Widgets require us to download supporting files from a 3rd party website. Click <a href='https://command:jupyter.enableLoadingWidgetScriptsFromThirdPartySource'>here</a> to enable this or click <a href='https://aka.ms/PVSCIPyWidgets'>here</a> for more information. (Error loading {0}:{1}).",
     "DataScience.enableCDNForWidgetsSetting": "Widgets require us to download supporting files from a 3rd party website. (Error loading {0}:{1}).",
     "DataScience.enableCDNForWidgetsButton": "Enable Downloads",
-    "DataScience.widgetScriptNotFoundOnCDNWidgetMightNotWork": "Unable to load a compatible version of the widget '{0}'. Expected behavior may be affected.",
+    "DataScience.widgetScriptNotFoundOnCDNWidgetMightNotWork": "Unable to find widget '{0}' version '{1}' from configured widget sources {2}. Expected behavior may be affected. More Info: 'https://aka.ms/PVSCIPyWidgets'",
```
[the remark this answers is not in the document]
Comment (rchiodo) [reply package.nls.json:492]: Yeah if they're not used anymore. Don't forget the other nls.json files too (that string may have been translated).

Comment (rchiodo) [inline package.nls.json:492]: I think you need to format the href link like the other examples. Otherwise the warning message won't be clickable.

**Comment by rchiodo** [comment]
You should probably add a news entry too.

**Review by rchiodo — APPROVED** [review]
Looks good. 

Maybe you can update this page with more info? How like the version may not be available on the CDN:

https://github.com/microsoft/vscode-jupyter/wiki/IPyWidget-Support-in-VS-Code-Python

**Review by IanMatthewHuff — APPROVED** [review]
Just to check looks like you might need to run Prettier on your changes. Do you have that extension installed?

**Pull request merged**